lib/open_project/scm/adapters/local_client.rb

```diff
@@ -28,6 +28,7 @@
 #++
 
 require 'open3'
+require 'find'
```

Comment:
No, it seems to be required:

`NameError: uninitialized constant Find`

when omitting that require. I have no clue though why this happens. It's a core module.